src/uwtools/resources/FV3Forecast.jsonschema

```diff
@@ -0,0 +1,167 @@
+{
+  "title": "FV3 Forecast config",
+  "description": "This document is to validate user-defined FV3 forecast config files",
+  "type": "object",
+  "properties": {
+    "platform": {
```

Comment:
Now that I'm taking another look at this, does it make sense for the FV3Forecast class to be validating information about the platform?